CondFormats/SiPixelTransient/src/T_EventSetup_SiPixelTemplateStore.cc

```diff
@@ -0,0 +1,19 @@
+// -*- C++ -*-
+//
+// Package:     CondFormats/SiPixelTransient
+// Class  :     T_EventSetup_SiPixelTemplateStore
+//
+// Implementation:
+//     [Notes on implementation]
+//
+// Original Author:  Christopher Jones
+//         Created:  Tue, 08 Aug 2023 15:33:48 GMT
+//
+
+// system include files
+
+// user include files
+#include "CondFormats/SiPixelTransient/interface/SiPixelTemplate.h"
+#include "FWCore/Framework/interface/eventsetuprecord_registration_macro.h"
```

Comment:
I believe this should be
```suggestion
#include "FWCore/Utilities/interface/typelookup.h"
```
which would then avoid adding the dependence on `FWCore/Framework` to this package.